AlsongDalsong/4-5/소스.c

```diff
@@ -0,0 +1,15 @@
+﻿#include <stdio.h>
+
+int main(void)
+{
+	int a = 0;
+	int i = 1;
+
+	printf("양의 정수를 입력하세요:");
+	scanf("%d", &a);
+
+	for (i; i <= a; i++)
```

Comment:
코드 위쪽의 `int i = 1`을 for의 초기화식에 넣어주세요.

```suggestion
	for (int i = 1; i <= a; i++)
```